meson.build

```diff
@@ -343,6 +344,7 @@ data_files = files(
 	'data/devices/logitech-MX-Anywhere2S.device',
 	'data/devices/logitech-MX-Anywhere-3.device',
 	'data/devices/logitech-MX-Vertical.device',
+	'data/devices/openinput.device',
```

Comment:
unrelated to this commit: we should use `install_subdir()` here which avoids having to name all `.device` files.

Reply:
Yes, I am not sure when that got added, meson didn't used to have it IIRC.

Reply:
ages ago. I remember looking it up for libwacom (same thing there) and eventually stopped bothering finding the exact version number. I think 0.22 or something like that, quick git log shows patches from 2014. Not sure if it was just a matter of me copy/pasting or not finding it when the inital meson support was added but eh.